cloudformation/transaction-storage.yaml

```diff
@@ -1,11 +1,11 @@
-AWSTemplateFormatVersion: '2010-09-09'
+AWSTemplateFormatVersion: "2010-09-09"
 Transform: AWS::Serverless-2016-10-31
 
 Resources:
   StorageBucket:
     Type: AWS::S3::Bucket
     Properties:
-      BucketName: !Join [ '-', [!Ref Prefix, !Ref Environment, storage ] ]
+      BucketName: !Join ["-", [!Ref Prefix, !Ref Environment, storage]]
```

Comment:
I like the spaces around the brackets, makes this horrible CF-syntax a bit more readable.

Reply:
the linting is auto doing this on save :(